contracts/StablecoinConverter.sol

```diff
@@ -17,24 +17,24 @@ contract StablecoinConverter is EpochTokenLocker {
         bool isSellOrder,
         uint32 validFrom,
         uint32 validUntil,
-        uint128 buyAmount,
-        uint128 sellAmount
+        uint128 priceNominator,
+        uint128 priceDenominator
     );
 
     event OrderCancelation(
         address owner,
         uint id
     );
 
-    //outstanding volume of an order is encoded as buyAmount or sellAmount depending on isSellOrder
     struct Order {
         uint16 buyToken;
         uint16 sellToken;
         uint32 validFrom;  // order is valid from auction collection period: validFrom inclusively
         uint32 validUntil;  // order is valid till auction collection period: validUntil inclusively
         bool isSellOrder;
-        uint128 buyAmount;
-        uint128 sellAmount;
+        uint128 priceNominator;
+        uint128 priceDenominator;
+        uint128 remainingAmount;
```

Comment:
Is this sell amount or buy amount or does it depend on isSellOrder?

Reply:
it is depended, 
but all the dependent logic is not yet implemented

Reply:
Should we call it remainingSellAmount for now then? Or at least comment that the side on which amount is on is dependent on the flag.